src/applications/personalization/profile-2/tests/unit-test-helpers.js

```diff
@@ -1,9 +1,37 @@
+import { waitForElementToBeRemoved } from '@testing-library/react';
+
 import { renderInReduxProvider } from 'platform/testing/unit/react-testing-library-helpers';
 
 import profile from 'applications/personalization/profile360/reducers';
 import connectedApps from 'applications/personalization/profile-2/components/connected-apps/reducers/connectedApps';
 import profileUi from '../reducers';
 
+// TODO: move this to platform/testing/unit/react-testing-library-helpers
+/**
+ * Returns a Promise that rejects if the element is removed from the DOM and
+ * resolves if the element is not removed. Uses DOM Testing Library's
+ * `waitForElementToBeRemoved()`.
+ *
+ * @param {} el - element that should remain in the DOM
+ * @param {} waitForOptions - options to pass to the `waitForElementToBeRemoved`
+ * call
+ */
+export async function elementNotRemoved(el, waitForOptions = {}) {
```

Comment:
Can we call this method `elementRemoved` so that we can swap the `resolve` and `reject` logic?